arm9/Makefile

```diff
@@ -10,8 +10,8 @@ INCDIRS := source source/common source/filesys source/crypto source/fatfs source
 INCLUDE := $(foreach dir,$(INCDIRS),-I"$(shell pwd)/$(dir)")
 
 ASFLAGS += $(SUBARCH) $(INCLUDE)
-CFLAGS  += $(SUBARCH) $(INCLUDE) -fno-builtin-memcpy -flto
```

Comment:
Not sure why we have to disable LTO, what error do you get when compiling?

Reply:
Okay this is.mainly for more control over the linking.

With LTO enabled it becomes impossible to move some functions to a different translation unit.